### PR TITLE
Fixes CASMPET-5013: update image refs in cray-metrics-server chart

### PR DESCRIPTION
**Pull request by bo-quan**

### CODE REVIEW

File: .github/workflows/charts-lint-test-scan-cron.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - cron: "1 0 * * *"
+  workflow_dispatch:
+jobs:
+  lint-test-scan:
+    uses: Cray-HPE/.github/.github/workflows/charts-lint-test-scan.yml@main
+    with:
+      ct-config: |
+        chart-dirs:
+          - kubernetes
+        validate-maintainers: false
```
Comment (zcrisler) [inline .github/workflows/charts-lint-test-scan-cron.yml:13]: Why is this false?

File: .github/workflows/charts-lint-test-scan.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  workflow_dispatch:
+jobs:
+  lint-test-scan:
+    uses: Cray-HPE/.github/.github/workflows/charts-lint-test-scan.yml@main
+    with:
+      lint-charts: ${{ github.event_name == 'pull_request' }}
+      ct-config: |
+        chart-dirs:
+          - kubernetes
+        validate-maintainers: false
```
Comment (zcrisler) [inline .github/workflows/charts-lint-test-scan.yml:16]: Why is this false?

File: kubernetes/cray-metrics-server/templates/deployment.yaml [hunk 3]
```diff
@@ -37,10 +38,12 @@ spec:
           containerPort: 4443
           protocol: TCP
         securityContext:
+          allowPrivilegeEscalation: false
```
Comment (zcrisler) [inline kubernetes/cray-metrics-server/templates/deployment.yaml:41]: Very nice!

**Review by zcrisler — APPROVED** [review]

File: .github/workflows/charts-lint-test-scan-cron.yml (hunk 1 above)
Comment (bo-quan) [reply .github/workflows/charts-lint-test-scan-cron.yml]: changed

**Pull request merged**